cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java

```diff
@@ -692,6 +695,9 @@ private boolean getPartitionLoad(HttpServletRequest request, HttpServletResponse
 
     String entriesString = request.getParameter(ENTRIES_PARAM);
     Integer entries = entriesString == null ? Integer.MAX_VALUE : Integer.parseInt(entriesString);
+    String topicToFilter = request.getParameter(TOPIC_PARAM);
+    String partitionToFilterString = request.getParameter(PARTITION_PARAM);
```

Comment:
Similar to regular expression support for `TOPIC_PARAM`, do you feel it'll be useful to support partition range for `PARTITION_PARAM`?